yarn.lock

```diff
@@ -13089,10 +13089,10 @@ parse-json@^5.0.0:
     json-parse-even-better-errors "^2.3.0"
```

Comment:
Let's remove the yarn.lock since there aren't any new packages being introduced.